dataplane/linux/endpoint_mgr_test.go

```diff
@@ -164,15 +164,19 @@ func chainsForIfaces(ifaceMetadata []string,
 	epMarkFromName := "cali-from-endpoint-mark"
 	epMarkSetOnePrefix := "cali-sm-"
 	epmarkFromPrefix := outPrefix[:6]
+	ipSetVXLANSourceHosts := "cali40all-vxlan-net"
+	ipSetAllHosts := "cali40all-hosts-net"
 	dropEncapRules := []iptables.Rule{
 		{
 			Match: iptables.Match().ProtocolNum(ProtoUDP).
+				DestIPSet(ipSetVXLANSourceHosts).
```

Comment:
I'm not completely following `DestIPSet` combined with `SourceHosts`. There's not a `SrcIPSet`?

Reply:
I'd assumed the ipSetVXLANSourceHosts was the set of hosts that support VXLAN - since I'm not sure it makes sense to think of source and destination separately for that.

Reply:
Yeah, I don't think it makes sense to think of it as source/dest. Maybe I can update the variable name here. 

The ipset that felix maintains has the variable name `IPSetIDAllVXLANSourceNets` and is actually a list of IP addresses of hosts (VXLAN parent device's IP address). I think that it was named as source nets because until now its only use was as a source match ipset.